### PR TITLE
Marked 1.11 as deprecated

### PR DESCRIPTION
**Pull request by sayboras**
### Description
Changes for https://github.com/weaveworks/eksctl/issues/1375. Only merge after 4 Nov 2019.

### Checklist
- [x] Code compiles correctly (i.e `make build`)
- [ ] Added tests that cover your change (if possible)
- [x] All unit tests passing (i.e. `make test`)
- [ ] Added/modified documentation as required (such as the `README.md`, and `examples` directory)
- [ ] Manually tested

### CODE REVIEW

**Comment by martina-if** [comment]
Hi @sayboras thank you for your PR. I would like to wait with this change until 1.11 is also marked as deprecated in the AWS console.

File: pkg/apis/eksctl.io/v1alpha5/types.go [hunk 1]
```diff
@@ -226,14 +226,13 @@ func SupportedRegions() []string {
 // https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html
 func DeprecatedVersions() []string {
 	return []string{
-		Version1_10,
+		Version1_10, Version1_11,
```
Comment (cPu1) [inline pkg/apis/eksctl.io/v1alpha5/types.go]: Minor nit:
```suggestion
		Version1_10,
		Version1_11,
```

Or
```golang
return []string{Version1_10, Version1_11}
```

**Comment by sayboras** [comment]
@martina-if as 0.8.0 is already released, the next release will be after 04 Nov, just want to check if we can merge this PR now.

**Comment by martina-if** [comment]
@sayboras great! thank you for the reminder. Can you rebase so I can approve it and merge it?

**Comment by sayboras** [comment]
@martina-if it's ready to merge now 💯

**Pull request merged**